lib/Joomla-CMS/ruleset.xml

```diff
@@ -2,29 +2,28 @@
 <ruleset name="Joomla-CMS">
 	<description>The Joomla CMS coding standard.</description>
 
-    <arg name="report" value="full"/>
-    <arg name="tab-width" value="4"/>
-    <arg name="encoding" value="utf-8"/>
-    <arg value="sp"/>
-    <arg name="colors" />
+	<arg name="report" value="full"/>
+	<arg name="tab-width" value="4"/>
+	<arg name="encoding" value="utf-8"/>
+	<arg value="sp"/>
+	<arg name="colors" />
+	<config name="php_version" value="70200"/>
+	<config name="installed_paths" value="../../joomla/cms-coding-standards/lib,../../joomla/coding-standards"/>
 
 	<!-- Exclude folders not containing production code -->
 	<exclude-pattern type="relative">^build/*</exclude-pattern>
-	<exclude-pattern type="relative">docs/*</exclude-pattern>
 	<exclude-pattern type="relative">cache/*</exclude-pattern>
-	<exclude-pattern type="relative">tmp/*</exclude-pattern>
+	<exclude-pattern type="relative">docs/*</exclude-pattern>
 	<exclude-pattern type="relative">logs/*</exclude-pattern>
+	<exclude-pattern type="relative">media/*</exclude-pattern>
+	<exclude-pattern type="relative">node_modules/*</exclude-pattern>
+	<exclude-pattern type="relative">tmp/*</exclude-pattern>
 
 	<!-- Exclude 3rd party libraries and Framework code. -->
-	<exclude-pattern type="relative">libraries/compat/password/*</exclude-pattern>
-	<exclude-pattern type="relative">libraries/fof/*</exclude-pattern>
-	<exclude-pattern type="relative">libraries/idna_convert/*</exclude-pattern>
 	<exclude-pattern type="relative">libraries/php-encryption/*</exclude-pattern>
-	<exclude-pattern type="relative">libraries/phputf8/*</exclude-pattern>
-	<exclude-pattern type="relative">libraries/simplepie/*</exclude-pattern>
 	<exclude-pattern type="relative">libraries/phpass/*</exclude-pattern>
 	<exclude-pattern type="relative">libraries/vendor/*</exclude-pattern>
```

Comment:
this line can be removed, redundant see line 31

Reply:
do we have another vendor folder somewhere? I'd actually would delete both line 31 and 32.

Reply:
oh sorry github commented the wrong line. 

Line 25 is already exclude on line 31.

Reply:
Yes, I understand, but if we have just one vendor folder, I don't see a reason for a more general rule like line 31. And since fof has been removed, I also don't see a reason to keep that exclusion as well.